chpl/chpl-service/src/main/java/gov/healthit/chpl/validation/surveillance/SurveillanceValidator.java

```diff
@@ -349,4 +355,26 @@ public void validateSurveillanceNonconformities(Surveillance surv) {
 			}
 		}
 	}
+	
+	public void validateSurveillanceAuthority(Surveillance surv) {
```

Comment:
1. I'm fine with the error message related to the ROLE
2. The UI will pass in one of three ROLEs: ROLE_ADMIN, ROLE_ACB_ADMIN, or ROLE_ACB_STAFF